domains/slovaks.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Documentation website for is-a.dev",
+  "repo": "https://github.com/is-a-dev/docs",
+  "owner": {
+    "username": "slovakians",
+    "email": "vipmodzteam@gmail.com"
+  },
+  "record": {
+    "CNAME": "slovakians.github.io/crvt"
```

CNAME can't contain slashes
```suggestion
    "CNAME": "slovakians.github.io"
```